runtime/src/accounts_db.rs

```diff
@@ -3805,11 +3805,17 @@ impl AccountsDb {
     where
         I: Iterator<Item = &'a Arc<AccountStorageEntry>>,
     {
-        let GetUniqueAccountsResult {
-            stored_accounts: stored_accounts_temp,
-            original_bytes,
-            store_ids,
-        } = self.get_unique_accounts_from_storages(stores);
+        let (
+            GetUniqueAccountsResult {
+                stored_accounts: stored_accounts_temp,
+                original_bytes,
+                store_ids,
+            },
+            measure,
```

Comment:
I think just give this a better name: `storage_read_elapsed`?

Odd to just call it `measure` when we have other `Measure`s in the same scope.